cmd/web3/main.go

```diff
@@ -995,6 +995,10 @@ func GenerateContract(ctx context.Context, contractType string, c *cli.Context)
 		if err != nil {
 			fatalExit(fmt.Errorf("Cloning finished with error: %v", err))
 		}
+		err = os.RemoveAll("lib/oz/.git")
+		if err != nil {
+			fatalExit(fmt.Errorf("Cannot cleanup .git dir in lib/oz: %v", err))
```

Comment:
Probably shouldn't happen, since we just created it, but maybe just a warning would be OK?

Reply:
Agree but it shouldn't fail at all and if it fails then something pretty bad is happening